src/component/common/FormTemplate/FormTemplate.tsx

```diff
@@ -144,12 +146,14 @@ const MobileGuidance = ({
 interface IGuidanceProps {
     description: string;
     documentationLink: string;
+    documentationLinkLabel?: string;
```

Comment:
Can we make the link label a required props "Learn more" is never a descriptive link label, so we should always provide more information. It would require changes in a lot of unrelated components, however, so it's probably more at home in a separate PR, but I'd love to hear some thoughts on this.

Reply:
Yeah, that would be great. A separate PR makes sense to me as well.

Reply:
Sweet; glad you agree 😄 I went ahead and made a [Trello ticket for it](https://trello.com/c/8qX1K0nC/524-documentation-links-are-not-descriptive)